src/serializers/ng-snapshot.ts

```diff
@@ -66,7 +66,7 @@ const print = (fixture: unknown, print: Printer, indent: Indent, opts: PluginOpt
       .map((node: VEDebugNode) => Array.from(node.renderElement.childNodes).map(print).join(''))
       .join(opts.edgeSpacing);
   }
-  const attributes = Object.keys(componentInstance);
+  const attributes = Object.keys(componentInstance).filter((key) => key !== '__ngContext__');
```

Comment:
Should we make an array or strings which refers to the attributes to remove in case later there are more?

Reply:
:+1: done

Reply:
messed it up; I guess I should have re-run the build before running test. sorry for inconveniences.